subgraph-js/src/schema-helper/buildSchemaFromSDL.ts

```diff
@@ -165,19 +165,17 @@ export function addResolversToSchema(
     const fieldMap = type.getFields();
 
     for (const [fieldName, fieldConfig] of Object.entries(fieldConfigs)) {
-      if (fieldName === "__resolveReference") {
+      if (fieldName === '__resolveReference') {
         type.extensions = {
           ...type.extensions,
           apollo: {
             ...type.extensions.apollo,
             subgraph: {
               ...type.extensions.apollo?.subgraph,
               resolveReference: fieldConfig,
-            }
+            },
           },
         };
-      } else if (fieldName.startsWith("__")) {
```

Comment:
I think we still want to support `__isTypeOf` (which is kind of the inverse `__resolveType`: instead of having an abstract type know how to probe objects and return a string, you can have each object type know how to return a bool)

Reply:
[29ae85c](https://github.com/apollographql/federation/pull/1757/commits/29ae85cf638df94d547c34417747ad0758206038)